srv/_modules/cluster.py

```diff
@@ -115,29 +128,50 @@
 
 #     for node in __pillar__["cluster"]["node_list"]:
 #         if __pillar__["cluster"][node]["is_primary"]:
-#             cmd = "multipath -ll | grep prio=50 -B2|grep mpath|sort -k2.2 | awk '{ print $1 }'"
+#             prio = 50
 #         else:
-#             cmd = "multipath -ll | grep prio=10 -B2|grep mpath|sort -k2.2 | awk '{ print $1 }'"
+#             prio = 10
+#         cmd = (
+#             "multipath -ll | grep prio={} -B2|".format(prio) +
```

Comment:
you might coonsider f-string: `f"multipath -ll | grep prio={prio} -B2|"`

Reply:
Done.